recipes/gperf/all/conanfile.py

```diff
@@ -56,6 +56,10 @@ def _configure_autotools(self):
 
     def _build_configure(self):
         with tools.chdir(self._source_subfolder):
+            try:
+                self.run("autogen.sh", run_environment=True)
+            except:
```

Comment:
You probably need to add an import at the top.
```suggestion
            except ConanException:
```